File: FrontAutomotiveAngular5/src/app/models/Ticket.ts
import { Officina } from './Officina';
import { Utente } from './Utente'; 

export class Ticket {
    private id: number;
    private data: string;
    private oggetto: string;
    private officina: Officina;
    private utente: Utente;


	constructor(id:number,data:string,oggetto:string,officina:Officina,utente:Utente) {
		this.id = id;
		this.data = data;
		this.oggetto = oggetto;
		this.officina = officina;
		this.utente = utente;
	}
        
}

File: src/main/java/it/contrader/automative/model/Ticket.java
package it.contrader.automative.model;

import java.io.Serializable;
import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.OneToOne;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor

@Entity
@Table(name = "ticket")
public class Ticket implements Serializable {

	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private int id;
	
	@Column
	private Date data;
	
	@Column
	private String oggetto;
	
	@ManyToOne(fetch=FetchType.LAZY)
	@JoinColumn(name = "idOfficina")
	private Officina officina;
	
	@ManyToOne(fetch=FetchType.LAZY)
	@JoinColumn(name = "idUtente")
	private Utente utente;

}
